examples/epd2in13bc.rs

```diff
@@ -0,0 +1,149 @@
+#![deny(warnings)]
+
+use embedded_graphics::{
+    fonts::{Font12x16, Font6x8, Text},
+    prelude::*,
+    primitives::{Circle, Line},
+    style::PrimitiveStyle,
+    text_style,
+};
+use embedded_hal::prelude::*;
+use epd_waveshare::{
+    color::*,
+    epd2in13bc::{Display2in13bc, Epd2in13bc},
+    graphics::{Display, DisplayRotation},
+    prelude::*,
+};
+use linux_embedded_hal::{
+    spidev::{self, SpidevOptions},
+    sysfs_gpio::Direction,
+    Delay, Pin, Spidev,
+};
+
+// activate spi, gpio in raspi-config
```

Comment:
Please leave a few lines of what this display is showing either here, next to `main`, or at the top to make it easier for new people to see what this example is giving to them (e.g. color display example,...)

Reply:
I added some comments to the example